src/radical/pilot/unit_manager.py

```diff
@@ -106,10 +106,11 @@ def __init__(self, session, scheduler=None):
                                     ns=session.uid)
         cfg.uid       = self.uid
         cfg.owner     = self.uid
-        cfg.sid       = session.uid
-        cfg.base      = session.cfg.base
-        cfg.path      = session.cfg.path
-        cfg.heartbeat = session.cfg.heartbeat
+        cfg.sid       = self._session.uid
```

Comment:
`self._session = session` is missing? 
```
(entk) hrlee@two:~/git/radical.pilot/examples$ python 00_getting_started.py

================================================================================
 Getting Started (RP version 0.90.8)
================================================================================

new session: [rp.session.two.hrlee.018247.0009]                                \
database   : [mongodb://rct:rct_test@two.radical-project.org/rct_test]        ok
read config                                                                   ok

--------------------------------------------------------------------------------
submit pilots

create pilot manager                                                          ok
submit 1 pilot(s)
        [local.localhost:1]
                                                                              ok

--------------------------------------------------------------------------------
submit units

caught Exception: 'UnitManager' object has no attribute '_session'
--------------
RADICAL Utils -- Stacktrace [10240] [MainThread]

hrlee    10240 26551  4 03:38 pts/40   00:00:00  |       \_ python 00_getting_started.py
hrlee    11278 10240  5 03:38 pts/60   00:00:00  |           \_ /bin/bash -i
Traceback (most recent call last):
File "00_getting_started.py", line 70, in <module>
umgr = rp.UnitManager(session=session)
File "/home/hrlee/venv3/entk/lib/python3.5/site-packages/radical/pilot/unit_manager.py", line 109, in __init__
cfg.sid       = self._session.uid
AttributeError: 'UnitManager' object has no attribute '_session'

```

Reply:
Ah, dammit - that's an ordering problem: the `self._session` assignment is done in the constructor of the base class (`rp.utils.Component`), but that is indeed called after.  I'll fix that.  Thanks.